### PR TITLE
chore: update lit to 3.0

### PR DESCRIPTION
**Pull request by TomMenga**
### Known issues
- [x] `@open-wc/lit-helpers` still has a dependency on lit 2.0 which causes the "Multiple version of lit..." warning

### CODE REVIEW

**Review by jeripeierSBB — COMMENTED** [review]
LGTM. can we not remove this.requestUpdate in setters now? -> https://lit.dev/blog/2023-10-10-lit-3.0/ 

> We now call requestUpdate() automatically for @property and @state decorated accessors where previously that was the setter's responsibility.

**Comment by TomMenga** [comment]
> LGTM. can we not remove this.requestUpdate in setters now? -> https://lit.dev/blog/2023-10-10-lit-3.0/
> 
> > We now call requestUpdate() automatically for @Property and @State decorated accessors where previously that was the setter's responsibility.

I thought that we would need to do that once we started using standard decorators but you might have a point. I might have misinterpreted it.
We also need to move the `@property` and `@state` decorators from the getters to the setters.

I'll test if it's true that the 'request update' is called automatically now

File: src/components/sbb-accordion/sbb-accordion.tsx [hunk 1]
```diff
@@ -102,6 +99,10 @@ export class SbbAccordion extends LitElement {
     this.addEventListener('will-open', (e) => this._closePanels(e), { signal });
   }
 
+  public override willUpdate(_changedProperties: Map<PropertyKey, unknown>): void {
```
Comment (jeripeierSBB) [inline src/components/sbb-accordion/sbb-accordion.tsx:102]: probably left unintentionally

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
A console log left and I suggest to use the following order: @property, setter, getter. I'ts more readable than getter, @property, setter.

**Pull request merged**